.circleci/config.yml

```diff
@@ -278,7 +313,6 @@ workflows:
           environment: ccq-uat
           context: laa-check-client-qualifies-uat
           requires:
-            - linters
```

Comment:
comment - hopefully you agree we should have 2 `linters` in this requires section.

Reply:
yes looks like duplication so good to remove it